preprocessing/filtered_innerouter_snsc_mousekidney.ipynb

```diff
@@ -210,90 +210,6 @@
     "bulk_adata = bulk_adata.transpose()"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    plt.xlabel('Num of Components')
</pre><p>This plot looks very strange -- is your sn.adata.X matrix transposed? Its just confusing that so little variation is explained. How many samples do you have? Can you plot the PCA plot to visually see why so little variation is explained? If it is this little variation, the plot should look almost perfectly spherical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/rnaseq_translator/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='133'/>